src/components/widgets/AutocompleteWidget/AutocompleteWidget.tsx

```diff
@@ -120,11 +108,25 @@ function AutocompleteWidget(props: AutocompleteWidgetProps) {
                     colorFirst={"primary"}
                     colorSecond={"success"}
                   />
-                  {!singleSuggestionRow && value.description ?
-                    <>
+                  {showApiSource && value.source && value.source !== "" &&
+                    <span 
+                      className="euiBadge"
+                      style={{
```

Comment:
Why use a different style than the breadcrumb widget?

Reply:
I could not get how to use it. Do you mean something like that?
`
<EuiBadge className="breadcrumb" color={props.colorFirst || "primary"}>
`

Reply:
Yes, I thought of a uniform style with the same CSS class.